### PR TITLE
Allow to disable daemonsets in load test

### PR DESCRIPTION
**Pull request by marseel**
**What type of PR is this?**

/kind feature
**What this PR does / why we need it**:
Currently, we don't have a way to disable daemonsets in load test. skip-steps does not help in this case, because there is one step that creates all objects. 
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: clusterloader2/testing/load/config.yaml [hunk 1]
```diff
@@ -194,13 +195,15 @@ steps:
       mediumServicesPerNamespace: {{DivideInt (AddInt $mediumDeploymentsPerNamespace 1) 2}}
       bigServicesPerNamespace: {{DivideInt (AddInt $bigDeploymentsPerNamespace 1) 2}}
 
+{{if $DISABLE_DAEMONSETS}}
```
Comment (mborsz) [inline clusterloader2/testing/load/config.yaml:198]: Shouldn't it be {{if not $DISABLE_DAEMONSETS}}??

Comment (marseel) [reply clusterloader2/testing/load/config.yaml]: yea, my bad. Done.

File: clusterloader2/testing/load/config.yaml [hunk 2]
```diff
@@ -373,7 +378,9 @@ steps:
     Instances:
      - Identifier: WaitForRunningDeployments
      - Identifier: WaitForRunningStatefulSets
+{{if not $DISABLE_DAEMONSETS}}
```
Comment (wojtek-t) [inline clusterloader2/testing/load/config.yaml:381]: is that needed? If there isn't any DS, it shouldn't matter at all, right?

Comment (marseel) [reply clusterloader2/testing/load/config.yaml:381]: Yes, we could skip this part. I personally don't see big difference between having it or not.

Comment (marseel) [reply clusterloader2/testing/load/config.yaml:381]: If you prefer to remove it, let me know and I will do it.

Comment (wojtek-t) [reply clusterloader2/testing/load/config.yaml:381]: The simpler the config is the better.
So if it works without this if too, I prefer removing it.

Comment (marseel) [reply clusterloader2/testing/load/config.yaml:381]: Let me double check to make sure it works without it

**Comment by wojtek-t** [comment]
/hold cancel

thanks!

**Pull request merged**